eZ/Publish/Core/Comparison/Engine/NoComparisonValueEngine.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Comparison\Engine;
+
+use eZ\Publish\API\Repository\Values\Content\VersionDiff\FieldType\NoComparisonResult;
+use eZ\Publish\SPI\Comparison\ComparisonData;
+use eZ\Publish\SPI\Comparison\ComparisonEngine;
+use eZ\Publish\SPI\Comparison\ComparisonResult;
+
+final class NoComparisonValueEngine implements ComparisonEngine
+{
+    public function compareFieldsData(ComparisonData $comparisonDataA, ComparisonData $comparisonDataB): ComparisonResult
+    {
+        return new NoComparisonResult();
```

Comment:
and here we could have ```return new NoComparisonEngine()```

Reply:
For me that would suggest Engine that do no comparison, and not result of that engine.

Reply:
Ok, you point is valid  👍